### PR TITLE
Make getFormulaFragmentId() a public function.

### PR DESCRIPTION
**Pull request by Laimiux**
## What

Making this into a public API.

### CODE REVIEW

**Comment by carrotkite** [comment]
### JaCoCo Code Coverage 97.16% :white_check_mark:
| Class | Covered | Meta | Status |
|:---|:---:|:---:|:---:|
| `com/instacart/formula/android/FragmentId` | 100% | 0% | :white_check_mark: |
| `com/instacart/formula/android/FormulaFragment` | 88% | 0% | :white_check_mark: |
| `com/instacart/formula/android/internal/FragmentFlowRenderView` | 72% | 0% | :white_check_mark: |
| `com/instacart/formula/android/internal/FragmentLifecycle` | 75% | 0% | :white_check_mark: |


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by Jawnnypoo — APPROVED** [review]
Is it expensive for it to generate a new one each time?

**Comment by Laimiux** [comment]
> Is it expensive for it to generate a new one each time?

What do you mean?

File: formula-android/src/main/java/com/instacart/formula/android/FragmentId.kt [hunk 1]
```diff
@@ -12,4 +16,14 @@ package com.instacart.formula.android
...
+
+/**
+ * Gets a [FragmentId] for a given [Fragment].
+ */
+fun Fragment.getFormulaFragmentId(): FragmentId {
+    return FragmentId(
+        instanceId = getFragmentInstanceId(),
+        key = getFragmentKey()
+    )
+}
```
[the remark this answers is not in the document]
Comment (Laimiux) [reply formula-android/src/main/java/com/instacart/formula/android/FragmentId.kt:29]: It's a tiny data class, should be completely fine.

**Pull request merged**